machine-runner/src/runner.test.ts

```diff
@@ -80,6 +80,7 @@ class Runner<
   private sub = Subscription.make<RegisteredEventsFactoriesTuple>()
   public machine
 
+  /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
we can leave it as is, or (for tests or internal code) disable this lint for the whole file — as you prefer